src_rebuild/premake_emulator.lua

```diff
@@ -57,9 +58,13 @@ project "PSX"
         links {
             "GL",
             "GLEW",
-            "openal", -- FIXME: is linux using openal-soft?
```

Comment:
I believe so, the [lib32-openal](https://www.archlinux.org/packages/multilib/x86_64/lib32-openal/) package on Arch Linux seems to contain the correct [upstream URL](https://github.com/kcat/openal-soft). The audio also seems to be working fine, as far as I can tell from my tests. I just needed to install `lib32-alsa-lib` and `lib32-pulseaudio` along with `lib32-openal`.